cpu/riscv_common/irq_arch.c

```diff
@@ -59,6 +68,9 @@ void riscv_irq_init(void)
     if (IS_ACTIVE(MODULE_PERIPH_PLIC)) {
         plic_init();
     }
+#ifdef MODULE_PERIPH_CLIC
```

Comment:
Not `if (IS_ACTIVE(MODULE_PERIPH_CLIC)) {` ?

Reply:
Shuffled the defines to the `clic.c` file to make this possible.